src/hotspot/share/adlc/formssel.cpp

```diff
@@ -2275,6 +2275,7 @@ bool OperandForm::is_bound_register() const {
   if (strcmp(name, "RegD") == 0) size = 2;
   if (strcmp(name, "RegL") == 0) size = 2;
   if (strcmp(name, "RegN") == 0) size = 1;
+  if (strcmp(name, "RegVectMask") == 0) size = 1;
```

Comment:
This might not work across architectures. @jatin-bhateja what are your thoughts on this?

Reply:
It doesn't have much usage apart from semantic checking for bound registers. 
But for correct ness a bounded opmask reg class for x86 has two registers.
Could not locate bounded reg class definitions for SVE predicate registers in aarch64.ad

Reply:
Yes, aarch64 does not need this for now. I can either revert this change (leaving it to you for future update if required) or update to size 2 for x86. Which do you prefer?

Reply:
I can see ADLC does parse the preprocessors in source sections, by adding a new #define SVE_ENABLE  1 in aarch64_sve.ad  and checking for the same similar to following handling may allow us to set size = 1 for SVE else 2.
https://github.com/openjdk/jdk/blob/master/src/hotspot/share/adlc/formssel.cpp#L2279

Reply:
That sounds good to me. Thanks @jatin-bhateja! Addressed in the new commit. Please let me know whether there are any other concerns.

Reply:
Thanks @nsjian , since adlc already consumes macros passed though command line your change looks good.